src/extensions/uptime.py

```diff
@@ -0,0 +1,15 @@
+import time as t; starttime = t.time()
+import arc
+import hikari
+import datetime as dt
+
+uptime_plugin = arc.GatewayPlugin("Blockbot Uptime")

```

Comment:
The other plugins use just `plugin` as the name, just to keep consistency across the codebase.

Reply:
sure ¯\(°_o)/¯